### PR TITLE
Fix/nct

### PR DESCRIPTION
**Pull request by mfshao**
Addressing
https://ctds-planx.atlassian.net/browse/PXP-6730
https://ctds-planx.atlassian.net/browse/PXP-6731
https://ctds-planx.atlassian.net/browse/PXP-6733
https://ctds-planx.atlassian.net/browse/PXP-6734
https://ctds-planx.atlassian.net/browse/PXP-6735
https://ctds-planx.atlassian.net/browse/PXP-6736
https://ctds-planx.atlassian.net/browse/PXP-6737

### CODE REVIEW

File: src/components/Introduction.jsx [hunk 1]
```diff
@@ -16,7 +16,11 @@ class Introduction extends Component {
     const buttonURL = `/study-viewer/${studyViewerConfig[0].dataType}`;
     return (
       <div className='introduction'>
-        <div className='h1-typo introduction__title'>{this.props.data.heading}</div>
+        <h1>
+          <div className='h1-typo introduction__title'>
+            {this.props.data.heading}
+          </div>
+        </h1>
```
Comment (paulineribeyre) [inline src/components/Introduction.jsx:23]: should this be merged to master? NIAID's comment about screen readers is relevant for all Commons

File: dev.html [hunk 2]
```diff
@@ -11,6 +11,8 @@
...
     <meta name="viewport" content="width=device-width" />
     <link href="https://fonts.googleapis.com/icon?family=Source+Sans+Pro" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Martel&display=swap" rel="stylesheet">
```
Comment (paulineribeyre) [inline dev.html:15]: what are these for?

Comment (mfshao) [reply dev.html:15]: oh that is for https://ctds-planx.atlassian.net/browse/PXP-6730

forget to include it in the PR description 🤦

File: src/components/layout/NavBar.jsx [hunk 3]
```diff
@@ -117,7 +117,7 @@ class NavBar extends Component {
                   <img
                     className='nav-bar__logo-img'
                     src='/src/img/logo.png'
-                    alt=''
+                    alt={components.homepageAltText || ''}
```
[the remark this answers is not in the document]
Comment (mfshao) [reply src/components/layout/NavBar.jsx:120]: yeah i can get another PR to master for this

File: src/components/Introduction.jsx (hunk 1 above)
Comment (mfshao) [reply src/components/Introduction.jsx:23]: same as above

File: src/components/layout/FooterNIAID.jsx [hunk 4]
```diff
@@ -57,16 +57,16 @@ const footerWebsiteLinks = [
...
   },
   {
     href: 'http://www.hhs.gov/',
-    text: 'Health and Human Services',
+    text: 'Health and Human Services ',
   },
   {
     href: 'http://www.usa.gov/',
-    text: 'USA.gov',
+    text: 'USA.gov ',
```
[the remark this answers is not in the document]
Comment (mfshao) [reply src/components/layout/FooterNIAID.jsx:69]: to get some space between text and the icon

this is ugly... i'll replece it with using antd `<Space>` component

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**